src/render.ts

```diff
@@ -133,6 +133,37 @@ export async function render(
       div.appendChild(storyResult);
       return true;
     }
+    case "Angular": {
+      const { props, component: StoryComponent, markup } = storyResult;
+      const { __decorate } = (await require("tslib"));
```

Comment:
do we really need this ?

Reply:
You are right, probably not as the render file is already TS.